README.md

```diff
@@ -93,6 +88,7 @@ batch script running GPU-enabled DFT-FE is below:
     MPICH_OFI_NIC_POLICY = NUMA
     HSA_FORCE_FINE_GRAIN_PCIE = 1 
     LD_LIBRARY_PATH = $LD_LIBRARY_PATH:$WD/env2/lib
+    MPICH_SMP_SINGLE_COPY_MODE=NONE
```

Comment:
Many of these variables are set in env.rc, so we don't need to explicitly set them here unless they are being changed from what env.rc did.